libs/dh/admin/feature-user-management/src/lib/drawer/dh-user-drawer.component.html

```diff
@@ -0,0 +1,34 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *transloco="let t; read: 'admin.userManagement.drawer'">
+  <watt-drawer #drawer size="large" (closed)="onClose()">
+    <watt-drawer-topbar>
+      <watt-badge type="success">{{ t("active") }}</watt-badge>
+    </watt-drawer-topbar>
+
+    <watt-drawer-actions>
+      <watt-button variant="secondary">{{ t("edit") }}</watt-button>
+    </watt-drawer-actions>
+
+    <watt-drawer-content>
+      <h1>{{ selectedUser?.name }}</h1>
```

Comment:
suggestion: according to Figma, the badge is placed next to the headline.

suggestion: also according to Figma, the headline should be `h2`.

Reply:
I know, but @PernilleChristiansen  showed my a new mockup with the status in the topbar. So I did that :-)

Reply:
Just for the record. The status badge is not placed next to the headline. It will be moved to the top bar as part of an UX alignment task.